### DISCUSSION TITLE
Cloudflare updates DDNS despite being up to date.

### DISCUSSION THREAD

**Discussion by jehlybean** [qdm12/ddns-updater#535] [Q&A]
I have GoDaddy, Google, and CloudFlare.  Only CloudFlare updates every single time there is a check.  Others just say "Up to date".

It works fine, but just wondering if there's a step I missed with CloudFlare?

**Comment by qdm12** [comment]
Are you running Cloudflare with the cloudflare proxy? If so, you need to enable the `proxied` option in your config.json
Otherwise ddns-updater detects an IP address every check period and will update every period.

**Reply by jehlybean** [reply]
> Are you running Cloudflare with the cloudflare proxy? If so, you need to enable the `proxied` option in your config.json Otherwise ddns-updater detects an IP address every check period and will update every period.

Tried this by placing `"proxied": "true"` on the bottom of the CF proxied options.  Now receiving a `ERROR json: cannot unmarshal string into Go struct field .proxied of type bool`

**Reply by qdm12** [reply]
Set it to `"proxied": true` (instead of `"true`)
